tests/suites/test_suite_x509parse.function

```diff
@@ -818,6 +818,39 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_X509_CRT_PARSE_C */
+void mbedtls_x509_get_name( char * rdn_sequence, int exp_ret )
+{
+    unsigned char *name;
+    unsigned char *p;
+    size_t name_len;
+    mbedtls_x509_name head = { 0 };
```

Comment:
Clang is not happy about that (I always forget about it) thus I think we have to use a `memset()` instead.